docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,26 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting server icon
+
+To get server icon URI from the client it has to be set by the server beforehand. Trying to read
+server icon uri before it was set will result in null value.
```

Comment:
Can we add one more line? 
For example:
To retrieve the server icon URI from the client, follow these steps:

Reply:
I rephrased the entire paragraph

Reply:
I think while rephrasing this line, the rephrase comment given for line no: 161 is not incorporated.
Therefore, can we combine and rephrase as:

Line no: 161: To retrieve the server icon URI from the client, the server must set the URI in advance. If you try to read the server icon URI before it is set, the result is a null value. 
Line no: 162: The following example explains how to retrieve the server icon URI on the client side:

Please verify and share your opinion.

Reply:
Right now this paragraph has the following form:

"Please follow the example presented below to retrieve the server icon URI on the client side.                                                           
If you try to retrieve the server icon URI before it was set, the result will be a null value. "

Please let me know what information is missing from it, because i think its equivalent to what you proposed.

Reply:
@m-michalski2 
Yes, I agree with you.
However, it is recommended to avoid using "please follow". Therefore, I rephrased it as:
https://github.com/Samsung/tizen-docs/pull/731#discussion_r290144090

Please verify and share your opinion.

Reply:
I replaced this paragraph with the version you provided. Please verify it looks okey now.

Reply:
Thank you very much for updating the changes.